### PR TITLE
feat(cdsctl): manage workflow label

### LINKED ISSUE #5023: cdsctl workflow label add / remove / list
add new sub-command

### PR DESCRIPTION
**Pull request by yesnault**
close #5023

Signed-off-by: Yvonnick Esnault <yvonnick@esnau.lt>

### CODE REVIEW

File: sdk/workflow.go [hunk 1]
```diff
@@ -253,7 +253,7 @@ type WorkflowNodeJobRunCount struct {
 // Label represent a label linked to a workflow
 type Label struct {
 	ID         int64  `json:"id" db:"id"`
-	Name       string `json:"name" db:"name"`
+	Name       string `json:"name" db:"name" cli:"label"`
```
Comment (fsamin) [inline sdk/workflow.go]:
```suggestion
	Name       string `json:"name" db:"name" cli:"label,key"`
```

File: engine/api/workflow.go [hunk 2]
```diff
@@ -249,7 +249,6 @@ func (api *API) postWorkflowLabelHandler() service.Handler {
 		key := vars["key"]
 		workflowName := vars["permWorkflowName"]
 		db := api.mustDB()
-		//u := getAPIConsumer(ctx)
```
Comment (richardlt) [inline engine/api/workflow.go]: To remove ?

File: cli/cdsctl/workflow_label.go [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+		{Name: "label"},
+	},
+}
+
+func workflowLabelDeleteRun(v cli.Values) error {
+	labelName := v.GetString("label")
+
+	modsWfs := []cdsclient.RequestModifier{}
+	modsWfs = append(modsWfs, func(r *http.Request) {
+		q := r.URL.Query()
```
Comment (richardlt) [inline cli/cdsctl/workflow_label.go]: Can create a func for that, cdsclient/interface.go

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#12638.0 ✘  
* Build
  * Build and Package All ✘  
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✘  
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#12646.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Pull request merged**